config/schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "doc": "The application environment.",
+    "format": ["production", "development", "test"],
+    "default": "development",
+    "env": "NODE_ENV"
+  },
+  "akismet_antispam_api_key": {
+    "doc": "akismet_antispam_api_key (ex. a1a11111aa11)",
+    "format": "String",
+    "default": "",
+    "env": "AKISMET_ANTISPAM_API_KEY"
+  },
+  "foo": {
```

Comment:
These are fake placeholder values.
IMO these should not be committed to the codebase.
I would create a schema.example.json or example-schema.json file instead and also make the fake values more meaningful, if possible, to provide strong clues to what they are being used for. And then I would .gitignore the config/schema.json so that each user is free to copy the example and change what they need to change.

Reply:
Agree with this :100:! `environments/development.json` and friends should be gitignored imho.

It hadn't occurred to me to do the same for `schema.json` (I didn't think custom config keys would be common except in a hard fork). I feel like I'm down for it, but can you also share an idea of when you imagine a forked shema would be helpful? Just asking to make sure I'm on the same page

Was thining a `make setup` target could